$NetBSD$

Do not override PKG_CONFIG_PATH to find pkgsrc dependencies.

--- Makefile.orig	2021-06-24 14:43:03.000000000 +0000
+++ Makefile
@@ -21,8 +21,8 @@ INCLUDEOPT:=-I$(CURDIR)/src -I$(CURDIR)/
 ifeq ($(shell pkg-config --version > /dev/null 2> /dev/null; echo $$?),0)
   HAS_PKGCONFIG:=yes
   # Fix for Mac OS X manual building with Homebrew
-  OLD_PKG_CONFIG_PATH:=$(PKG_CONFIG_PATH)
-  OVERRIDE_PC_PATH:=PKG_CONFIG_PATH=$(OLD_PKG_CONFIG_PATH):/usr/local/lib/pkgconfig:/opt/X11/lib/pkgconfig
+  # OLD_PKG_CONFIG_PATH:=$(PKG_CONFIG_PATH)
+  # OVERRIDE_PC_PATH:=PKG_CONFIG_PATH=$(OLD_PKG_CONFIG_PATH):/usr/local/lib/pkgconfig:/opt/X11/lib/pkgconfig
 else
   HAS_PKGCONFIG:=no
 endif
